thread/common/thrprof.c

```diff
@@ -121,14 +121,32 @@ omrthread_get_cpu_time_ex(omrthread_t thread, int64_t *cpuTime)
 #if defined(OMR_OS_WINDOWS) && !defined(BREW)
 	{
 		intptr_t ret = 0;
-		FILETIME creationTime, exitTime, kernelTime, userTime;
-		int64_t totalTime;
+		FILETIME creationTime;
+		FILETIME exitTime;
+		FILETIME kernelTime;
+		FILETIME userTime;
+		int64_t totalTime = 0;
+
+		memset(&creationTime, 0, sizeof(creationTime));
+		memset(&exitTime, 0, sizeof(exitTime));
+		memset(&kernelTime, 0, sizeof(kernelTime));
+		memset(&userTime, 0, sizeof(userTime));
 
 		/* WARNING! Not supported on Win95!  Need to test to ensure this fails gracefully */
 
 		if (GetThreadTimes(thread->handle, &creationTime, &exitTime, &kernelTime, &userTime)) {
-			totalTime = ((int64_t)kernelTime.dwLowDateTime | ((int64_t)kernelTime.dwHighDateTime << 32))
-						+ ((int64_t)userTime.dwLowDateTime | ((int64_t)userTime.dwHighDateTime << 32));
+			ULARGE_INTEGER kernelTimeU64;
+			ULARGE_INTEGER userTimeU64;
+			memset(&kernelTime, 0, sizeof(kernelTime));
+			memset(&userTime, 0, sizeof(userTime));
```

Comment:
These initialize the wrong variables.
```c
			memset(&kernelTimeU64, 0, sizeof(kernelTimeU64));
			memset(&userTimeU64, 0, sizeof(userTimeU64));
```
Also a blank line between the declarations and these calls would be welcome (see e.g. line 129).

Reply:
> a blank line between the declarations and these calls would be welcome

Please add that blank line (for consistency with other parts of this change).

Reply:
Sorry I forgot about that.